README.md

```diff
@@ -154,6 +154,13 @@ These error messages can also be amended via the Adapt Authoring Tool - but must
 "CLIENT_COULD_NOT_CONNECT": "La connexion avec la plate-forme de formation n'a pas pu être établie."
 ```
 
+### Print completion information from LMS data
+Using the lms suspend data from a different course:
```

Comment:
```suggestion
If you have a course where learners are reporting completion problems, it can often be useful to check the stored suspend data to see if they are simply missing something out. As the relevant part of the suspend data is no longer in 'human-readable' format, Spoor v3.8.0 includes a `printCompletionInformation` function that translates this into into a more readable string of 1s and 0s which you can then match to the course's 'tracking ids' to see which bits of the course the learner hasn't completed.

To do this, run any course that uses Spoor v3.8.0 (or better) and the following into the browser console (naturally, you need to replace the `suspendData` shown below with the one from the course you're trying to debug...)
```